packages/frontend/components/Navigation/Navigation.tsx

```diff
@@ -21,32 +21,39 @@ export enum Page {
 interface NavItem {
   page: Page
   iconPath: string
+  text: string
 }
 
 const navItems: NavItem[] = [
   {
     page: Page.HOME,
     iconPath: 'navigation/HomeIcon.svg',
+    text: 'Home',
   },
   {
     page: Page.MESSAGES,
     iconPath: 'navigation/MessageIcon.svg',
+    text: 'Messages',
   },
   {
     page: Page.FANART,
     iconPath: 'navigation/ImageIcon.svg',
+    text: 'Fanarts',
   },
   {
     page: Page.CLIPS,
     iconPath: 'navigation/VideoIcon.svg',
+    text: 'Clips',
   },
   {
     page: Page.SOUNDBOARD,
     iconPath: 'navigation/SoundIcon.svg',
+    text: 'Soundboard',
   },
   {
     page: Page.CREDITS,
     iconPath: 'navigation/InfoIcon.svg',
+    text: 'Credits',
```

Comment:
You could just use the page as the hover text, as i've assigned the enum strings. (fanart doesnt have a plural btw)

Reply:
Oh yeah lol I can do that I guess